src/window.rs

```diff
@@ -850,10 +852,14 @@ impl Default for CursorIcon {
     }
 }
 
+/// Fullscreen mode to be set on certain output.
```

Comment:
```suggestion
/// Fullscreen modes.
```

The certain output stuff isn't really true since it might not be a specific output and Exclusive takes a VideoMode instead?